package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "health-icons",
+  "name": "healthicons",
```

Comment:
NPM uses `package.json#name` for [unscoped packages](https://docs.npmjs.com/creating-and-publishing-unscoped-public-packages).